src/library/scala/Predef.scala

```diff
@@ -249,7 +249,7 @@ object Predef extends LowPriorityImplicits {
    *           }}}
    *  @group utilities
    */
-  @inline def locally[T](x: T): T = x
+  @inline def locally[T](@deprecatedName("x") x: T): T = x
```

Comment:
I almost expected `def locally[T](@deprecatedName x: T)` to work.

Reply:
By any other name, it would be a code smell.